x/auth/keeper/migrations.go

```diff
@@ -0,0 +1,146 @@
+package keeper
+
+import (
+	"fmt"
+
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+
+	"github.com/cosmos/cosmos-sdk/x/auth/vesting/exported"
+
+	"github.com/gogo/protobuf/grpc"
+	"github.com/gogo/protobuf/proto"
+	abci "github.com/tendermint/tendermint/abci/types"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+)
+
+const (
+	delegatorDelegationPath = "/cosmos.staking.v1beta1.Query/DelegatorDelegations"
+	balancesPath            = "/cosmos.bank.v1beta1.Query/AllBalances"
+)
+
+// Migrator is a struct for handling in-place store migrations.
+type Migrator struct {
+	keeper      AccountKeeper
+	queryServer grpc.Server
+}
+
+// NewMigrator returns a new Migrator.
+func NewMigrator(keeper AccountKeeper, queryServer grpc.Server) Migrator {
+	return Migrator{keeper: keeper, queryServer: queryServer}
+}
+
+// Migrate1to2 migrates from version 1 to 2.
```

Comment:
I would elaborate a bit more on what's happening here. Doesn't have to be too verbose though.

Reply:
Yes, we're just shushing the linter right now.

Proper comments are bound to happen as soon as we get everything good code-wise.

Reply:
@gsora now maybe is the moment to improve the comments here.